system/modules/isotope/dca/tl_iso_gallery.php

```diff
@@ -218,7 +218,7 @@
             'label'                 => &$GLOBALS['TL_LANG']['tl_iso_gallery']['zoom_size'],
             'exclude'               => true,
             'inputType'             => 'imageSize',
-            'options'               => $GLOBALS['TL_CROP'],
+            'options'               => \System::getImageSizes(),
```

Comment:
Please use the options_callback so this stuff is lazy loaded :) The data is loaded in the FE everytime a Model instance is created and thus relevant for performance :)